tests/libvmi/vmi.go

```diff
@@ -48,12 +47,13 @@ func New(opts ...Option) *kvirtv1.VirtualMachineInstance {
 
 // randName returns a random name for a virtual machine
 func randName() string {
-	return "testvmi" + "-" + rand.String(5)
+	const posixLen = 5
```

Comment:
why 5 ? is there a name this const can have that makes it clear where this 5 comes from ? 

Might be just me, but it's not clear to me.

Reply:
I think it is just a length that has enough probability to be unique in the system.

I renamed it now to `randomPostfixLen` to emphasize it is about the random part len.

But lets be practical here, this is a one away from its usage, it does not need a full name to make it clear for what it is used. The context matters, rules for proper names are not really needed for localize (a few lines) scoped code.

Reply:
I think it's OK - was just trying to understand what that number represented. Adding the suffix `random` or `arbitrary` makes it explicit that I shouldn't bother trying to make sense of it.